sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -679,3 +679,30 @@ case class TruncateTable(
     partitionSpec: Option[TablePartitionSpec]) extends Command {
   override def children: Seq[LogicalPlan] = child :: Nil
 }
+
+/**
+ * The logical plan of the CACHE TABLE command.
+ */
+case class CacheTable(
+    child: LogicalPlan,
```

Comment:
After more thought, I think CACHE TABLE is not a DDL command that needs to interact with catalogs, and it doesn't need a v2 version.

The current problem is that `CacheTableCommand` only takes v1 table identifier and can't cache v2 tables with n part name. Maybe we can fix `CacheTableCommand` directly?

Reply:
I updated `CacheTableCommand` and `uncacheTable` to support multiparts name (and not resolving the identifier). Please check what you think about the new approach. Thanks.

Reply:
Also, since it's not resolving to catalogs, we should move it out of `ResolveSessionCatalog`?